dist/redhat/scylla.spec

```diff
@@ -148,6 +151,7 @@ ln -sfT /etc/scylla /var/lib/scylla/conf
 %ghost %config /etc/systemd/system/var-lib-systemd-coredump.mount
 %ghost /etc/systemd/system/scylla-cpupower.service
 %ghost %config /etc/systemd/system/var-lib-scylla.mount
+%{_sysconfdir}/bash_completion.d/nodetool-completion
```

Comment:
note to myself, as a follow-up change, might want to rename it to `nodetool`. looks better this way. see the output of

```console
ls /etc/bash_completion.d/
```
for more examples.